### PR TITLE
Created DigitalOcean driver

### PR DESCRIPTION
**Pull request by robertsheehy-wf**
## What

Created a driver for using DigitalOcean as a beacon provider.
### Create Read-Only API Token

screen shot 2015-02-09 at 9 22 32 am
### Copy Token

screen shot 2015-02-09 at 9 22 41 am
### When Creating Beacon Droplet Include Token in "User Data"

screen shot 2015-02-09 at 9 19 35 am

### CODE REVIEW

**Comment by theMagicalKarp** [comment]
@lighthouse/owners

**Comment by cmbrose** [comment]
+1

File: drivers/ocean/ocean.go [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+        oceanToken = string(body)
+    }
+
+    return oceanToken
+}
+
+func IsApplicable() bool {
+    request, _ := http.NewRequest("GET",
+        "http://169.254.169.254/metadata/v1/", nil)
```
Comment (nickmonad) [inline drivers/ocean/ocean.go:81]: Does this IP remain fixed?

Comment (robertsheehy-wf) [inline drivers/ocean/ocean.go:81]: Yup, https://developers.digitalocean.com/metadata/.  It'a actually the same address for [ec2](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-metadata.html) & [gce](https://cloud.google.com/compute/docs/metadata) metadata.

**Comment by nickmonad** [comment]
+1

**Comment by chfogerty** [comment]
+1

**Pull request merged**